app/models/scsb/partner_updates/update.rb

```diff
@@ -51,6 +51,18 @@ def process_partner_updates(files:, file_prefix: 'scsb_update_')
         end
       end
 
+      def self.process_record(record)
```

Comment:
This appears to be the same method as the one starting line 66 but called "self.", if we don't need the other one any longer we can remove.

Reply:
Done - the incremental update was still using the instance method, so just added that change to the refactor.